shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2018 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_PLATFORM_DARWIN_IOS_FRAMEWORK_SOURCE_FLUTTERPLATFORMVIEWS_INTERNAL_H_
+#define FLUTTER_SHELL_PLATFORM_DARWIN_IOS_FRAMEWORK_SOURCE_FLUTTERPLATFORMVIEWS_INTERNAL_H_
+
+#include "flutter/fml/platform/darwin/scoped_nsobject.h"
+#include "flutter/shell/common/shell.h"
+#include "flutter/shell/platform/darwin/ios/framework/Headers/FlutterBinaryMessenger.h"
+#include "flutter/shell/platform/darwin/ios/framework/Headers/FlutterChannels.h"
+#include "flutter/shell/platform/darwin/ios/framework/Headers/FlutterPlatformViews.h"
+
+namespace shell {
+
+class FlutterPlatformViewsController {
+ public:
+  FlutterPlatformViewsController(NSObject<FlutterBinaryMessenger>* messenger);
+
+  void RegisterViewFactory(NSObject<FlutterPlatformViewFactory>* factory, NSString* factoryId);
+
+ private:
+  fml::scoped_nsobject<FlutterMethodChannel> channel_;
+  std::map<std::string, fml::scoped_nsobject<NSObject<FlutterPlatformViewFactory>>> factories_;
+  std::map<long long, fml::scoped_nsobject<NSObject<FlutterPlatformView>>> views_;
```

Comment:
Please replace this with one of the types in `stdint.h`

Reply:
done